lesson_19/lesson_19_turcu_alexandru/context_manager.py

```diff
@@ -0,0 +1,55 @@
+'''Challenge
+Implement a context manager called just_some_exceptions that will handle KeyError, IndexError by printing a message and
+let any other exceptions propagate outside of the context manager.
+
+Implement the context manager by using both approaches: by using class and by using @contextmanager
+
+Write examples that prove the functionality for all the handled exceptions and for one exception that is not handled.'''
+import contextlib
+
+
+@contextlib.contextmanager
+def some_exceptions(context):
```

Comment:
it is ok to have an argument to your context manager. Remember that what you are yielding, in this case, the `context` is what is attributed to the variable after as
```
whit some_exceptions(context) as context 
```

when you use the context manager you don't have anything after `as`

Reply:
You mean "with SomeException() as context: " right? So its not ideal to have an argument to my context manager. 
Thank you, I will be better in the future.